### ISSUE TITLE
Property interpolation is not working

### ISSUE THREAD

**Issue by jdeppe-pivotal** [pivotal-cf/java-cfenv#70]
Hi,

I have an `application-cloud.properties` file with the following lines:
```
spring.datasource.url=#{ cfEnv.findCredentialsByTag('datasource').get('url') }
spring.datasource.username=#{ cfEnv.findCredentialsByTag('datasource').get('username') }
spring.datasource.password=#{ cfEnv.findCredentialsByTag('datasource').get('password') }
```
and a `cfEnv` bean defined as:
```
	@Bean
	public CfEnv cfEnv() {
		return new CfEnv();
	}
```

When I `cf push` my app, the log seems to indicate these values are not being interpolated correctly as I see this:
```
   2019-08-29T06:52:28.03-0700 [APP/PROC/WEB/0] OUT 2019-08-29 13:52:28.032 DEBUG 27 --- [           main] com.zaxxer.hikari.HikariConfig           : jdbcUrl.........................#{ cfEnv.findCredentialsByTag('datasource').get('url') }
   2019-08-29T06:52:28.03-0700 [APP/PROC/WEB/0] OUT 2019-08-29 13:52:28.034 DEBUG 27 --- [           main] com.zaxxer.hikari.HikariConfig           : username........................"#{ cfEnv.findCredentialsByTag('datasource').get('username') }"
```
... and then there is subsequent breakage.

My application dependencies include the following:
```
    implementation('io.pivotal.cfenv:java-cfenv-boot:1.1.1.RELEASE')
    implementation('org.springframework.boot:spring-boot-starter-cloud-connectors')
    implementation('org.springframework.boot:spring-boot-starter-data-jpa')
    implementation('org.springframework.boot:spring-boot-starter-web')
    implementation('org.springframework.boot:spring-boot-starter-actuator')
```
What am I missing in order for these values to be interpolated correctly?

In truth, I have a somewhat bespoke environment consisting of https://github.com/cloudfoundry-incubator/cfdev and a user-provided HSQLDB CF service (running on my laptop since I'm testing locally). The service is configured with the following tags `relational, database, hsql, hsqldb, datasource`.

**Comment by maximusfloydus** [comment]
@jdeppe-pivotal  Did you ever find a solution for this?  It was recommended I used this method via #87 by @dyroberts.  I also couldn't get the values to resolve.

**Comment by dyroberts** [comment]
Fixed in 6850d22
